lib/proximity.js

```diff
@@ -190,6 +190,36 @@ var Controllers = {
       }
     }
   },
+  HCSR04I2C: {
+    initialize: {
+      value: function(opts, datahandler) {
+        var BYTES_TO_READ = 0x02;
+        var address = opts.address || 0x27;
+        var msToNextRead = 90;
+
+        opts.address = address;
+
+        // set up IO connection:
+        this.io.i2cConfig(opts);
+
+			  var read = function() {
+				  setTimeout(function() {
+					  this.board.io.i2cReadOnce(address, BYTES_TO_READ, function(data) {
```

Comment:
Why `i2cReadOnce`? `i2cRead` is always more efficient—can that be done? Presumably the delays between actual reading on the backpack can behandled there. Even if the continuous read gets whatever is in the register twice in a row?